Add Chartbrew to the data visualization integrations page

razvanilin opened the pull request:
## Summary

Added Chartbrew to the data visualization integrations page. Also added an entry for Chartbrew along with its logo in the main Integrations page

- Issue: #3449 

## Checklist
- [x] Delete items not relevant to your PR
- [x] URL changes should add a redirect to the old URL via https://github.com/ClickHouse/clickhouse-docs/blob/main/docusaurus.config.js
- [x] If adding a new integration page, also add an entry to the integrations list here: https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/integrations/index.mdx

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ClickHouse/clickhouse-docs?pullRequest=3450) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ClickHouse/clickhouse-docs?pullRequest=3450) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ClickHouse/clickhouse-docs?pullRequest=3450) it.</sub>

gingerwizard commented:
Hi @razvanilin we welcome this PR and thanks. We're delaying merging as we want to reorg this section of the docs next week to clearly show this is a community supported integration and not official - we've had some confusion arise here with our users. We'll make sure its then merged.

razvanilin commented:
Thanks @gingerwizard and no worries about the delay! Let me know if there is anything I can change from my side. Happy to help

gingerwizard commented:
Hi @razvanilin few things:

1. Can you merge with main and update to use the new image component. Details https://github.com/ClickHouse/clickhouse-docs/pull/3526/files
2. Import the community maintained badge and add - example [here](https://github.com/ClickHouse/clickhouse-docs/blob/178fd1957f83968cad536b462ec9b531488b7e2a/docs/integrations/data-visualization/deepnote.md?plain=1#L15-L19)

gingerwizard reviewed (changes requested):
see comment

razvanilin commented:
Hey @gingerwizard I tried merging with main, also create a clean branch with the latest `main` but I always get this error when I try to build the project:

```
[ERROR] Loading of version failed for version current

[ERROR] Error: Unable to build website for locale en.
    at tryToBuildLocale (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:78:15)
    at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:34:9
    ... 4 lines matching cause stack trace ...
    at async file:///Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/bin/docusaurus.mjs:44:3 {
  [cause]: Error: Doc with ID ru/index wants to display sidebar russia but a sidebar with this name doesn't exist
      at Object.getDocNavigation (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/sidebars/utils.js:124:19)
      at addNavData (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:162:42)
      at Array.map (<anonymous>)
      at addDocNavigation (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:190:37)
      at doLoadVersion (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:183:55)
      at async loadVersion (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:193:28)
      at async Promise.all (index 0)
      at async Object.loadContent (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:201:33)
      at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:39:23
      at async Promise.all (index 0)
      at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:140:25
      at async loadSite (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/server/site.js:144:45)
      at async buildLocale (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js:28:18)
      at async runBuildLocaleTask (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:93:5)
      at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:74:13
      at async tryToBuildLocale (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:70:9)
      at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:34:9
      at async mapAsyncSequential (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/utils/lib/jsUtils.js:21:24)
      at async Command.build (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/build/build.js:33:5)
      at async Promise.all (index 0)
      at async runCLI (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/commands/cli.js:56:5)
      at async file:///Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/bin/docusaurus.mjs:44:3
}
```

Running the development fails as well (I'm assuming because the build failed):

```
yarn run v1.22.22
warning ../package.json: No license field
$ docusaurus start
[INFO] Starting the development server...
[ERROR] Loading of version failed for version current

[ERROR] Error: Doc with ID ru/index wants to display sidebar russia but a sidebar with this name doesn't exist
    at Object.getDocNavigation (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/sidebars/utils.js:124:19)
    at addNavData (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:162:42)
    at Array.map (<anonymous>)
    at addDocNavigation (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:190:37)
    at doLoadVersion (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:183:55)
    at async loadVersion (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:193:28)
    at async Promise.all (index 0)
    at async Object.loadContent (/Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/index.js:201:33)
    at async /Users/razvanilin/Documents/devStuff/clickhouse-docs/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:39:23
    at async Promise.all (index 0)
[INFO] Docusaurus version: 3.7.0
Node version: v20.18.3
```

I made sure I ran `yarn install` before the build command. Is there something that I'm missing or some new requirements for building the project?

Blargian commented:
@razvanilin We made several large changes to the repo recently, one of which was how we handle translations. If you have `ru` and `zh` folders under `docs` could you delete them and then run `yarn clear` and `yarn build` and see if that helps?

razvanilin commented:
@Blargian that did the trick!

I added the requested changes now. Made sure I merged with `main`, replaced all <img> tags with <Image> and added the community badge to the Chartbrew page.

Let me know if I missed anything. Happy to update
